cargo-near/Cargo.toml

```diff
@@ -27,7 +27,7 @@ schemars = "0.8"
 
 [dependencies.near-abi]
 git = "https://github.com/near/near-abi-rs"
-rev = "4844ef4ff3e5cf3c3a8aa6565c3195a2339a345f"
+rev = "983346e8ce2a8304645a92ba7735f9a6eacceda7"
 features = ["__chunked-entries"]
```

Comment:
Can we use `0.1.0-pre.0`? https://crates.io/crates/near-abi/0.1.0-pre.0

Reply:
has already been updated.

Reply:
oh, oops, must have been looking at an outdated commit